autoload/wispath.vim

```diff
@@ -70,11 +70,16 @@ enddef
 
 export def GetCompletion(target_path: string, cursor_col: number, in_cmdline: bool): list<any>
   var completions: list<string>
-  if IsInCmdwin()
-    completions = EvalInAlterWin(function('getcompletion', [target_path, 'file']))
-  else
-    completions = getcompletion(target_path, 'file')
-  endif
+  try
+    if IsInCmdwin()
+      completions = EvalInAlterWin(function('getcompletion', [target_path, 'file']))
+    else
+      completions = getcompletion(target_path, 'file')
+    endif
+  catch /^Vim\%((\a\+)\)\=:\(E219\|E220\):/
```

Comment:
```suggestion
  catch /^Vim\%((\a\+)\)\=:\%(E219\|E220\):/
```

Using grouping without capturing is bit faster. (This doesn't much matter here though.)
https://vim-jp.org/vimdoc-ja/pattern.html#/\%(\)